torch_xla/csrc/ops/xla_ops.cpp

```diff
@@ -5,6 +5,9 @@ namespace ir {
 namespace ops {
 
 const OpKindWrapper xla_all_to_all("xla::all_to_all");
+const OpKindWrapper xla_amp_foreach_non_finite_check_and_unscale(
+    "xla::amp_foreach_non_finite_check_and_unscale");
+const OpKindWrapper xla_amp_update_scale("xla::amp_update_scale");
```

Comment:
I think we should move them to pytorch since these ops are not xla specified.

Reply:
I agree.